CONTRIBUTING.md

```diff
@@ -240,15 +236,17 @@ There are a few main venues of communication:
 * For more organizational concerns, the [mailing
   list](http://www.haskell.org/mailman/listinfo/cabal-devel) is used.
 
-* Many developers idle on `#hackage` on `irc.freenode.net` ([archives](http://ircbrowse.net/browse/hackage)).  `#ghc` ([archives](http://ircbrowse.net/browse/ghc)) is also a decently good bet.
+* Many developers idle on `#hackage` on https://libera.chat. The `#ghc` channel is also a decently good bet.
+  * You can join the channel using a web client, even anonymously: https://kiwiirc.com/nextclient/irc.libera.chat/?nick=Guest%7C?#hackage
```

Comment:
We should link to the [official webclient](https://web.libera.chat/#hackage) rather than kiwi.

Reply:
yeah, will change